console_windows.go

```diff
@@ -30,6 +30,9 @@ var (
 )
 
 func (m *master) initStdios() {
+	// Note: We discard console mode warnings, because in/out can be redirected.
```

Comment:
```suggestion
	// Note: We discard console mode warnings, because in/out can be redirected.
	//
	// TODO: Investigate opening CONOUT$/CONIN$ to handle this correctly
```

Reply:
Done!